cli/internal/login/link.go

```diff
@@ -167,13 +167,17 @@ func (l *link) run() error {
 	}
 	isUser := (chosenTeamName == userResponse.User.Name) || (chosenTeamName == userResponse.User.Username)
 	var chosenTeam client.Team
-	if !isUser {
+	var accountID string
```

Comment:
switched to `teamID`, and while doing another reading pass, moved the call to `apiClient.SetTeamID()` to cover the single user case also, since that matches what we use for `apiClient.IsLoggedIn()`.